### DISCUSSION TITLE
Possibility to port chart to excel?

### DISCUSSION THREAD

**Discussion by BaldHawk** [Kanaries/pygwalker#370] [General]
I just wanted to ask whether we can export the chart as xlsx (not csv) and then edit the text on the X-axis and Y-axis? It would be great if we can do so. Any alternative method with which we can achieve this (if not possible otherwise)?

**Comment by longxiaofei** [comment]
Did you need to export datas required for render the chart? or, only chart?

Export to png and embed png into your excel, this is currently possible.

The feature of editing axis text is not yet implemented, but it’s already on our todo list.

**Reply by BaldHawk** [reply]
Yes I was asking for the feature which you said is in your to-do list. Thanks. Appreciate it. Btw the tool is very good and very handy.

**Reply by longxiaofei** [reply]
Thanks for using pygwalker. The Kanareis team will continuously improve pygwalker to make it more user-friendly.

**Reply by prateeksaurabh** [reply]
Was this feature ever implemented or was this dropped
